<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uorPageSetup</class>
 <widget class="QDialog" name="uorPageSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>279</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>385</width>
    <height>279</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>385</width>
    <height>279</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>245</y>
     <width>167</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="title">
    <string>Paper</string>
   </property>
   <widget class="QComboBox" name="m_cbPaperSize">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>20</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="m_llPaperSize">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Paper size</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="title">
    <string>Orientation</string>
   </property>
   <widget class="QRadioButton" name="m_rbPortrait">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Portrait</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_rbLandscape">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>20</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>landscape</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>230</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Fields (mm.)</string>
   </property>
   <widget class="QLabel" name="m_llFieldsLeft">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_leFieldsLeft">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>19</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>99</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="cursorPosition">
     <number>2</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_leFieldsTop">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>49</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>99</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="cursorPosition">
     <number>2</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="m_llFieldsTop">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>50</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Top</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_leFieldsBottom">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>49</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>99</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="cursorPosition">
     <number>2</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="m_llFieldsRight">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="m_llFieldsBottom">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bottom</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_leFieldsRight">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>19</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>99</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="cursorPosition">
     <number>2</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>11</y>
     <width>130</width>
     <height>138</height>
    </rect>
   </property>
   <property name="title">
    <string>View</string>
   </property>
   <widget class="QLabel" name="m_llPageViewP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="uoReport.qrc">:/image/portrait.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="m_llPageViewL">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>30</y>
      <width>120</width>
      <height>90</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="uoReport.qrc">:/image/landscape.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="title">
    <string>Running title (mm.)</string>
   </property>
   <widget class="QLineEdit" name="m_leRTitleBottom">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>18</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>99</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="cursorPosition">
     <number>2</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_leRTitleTop">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>18</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>99</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="cursorPosition">
     <number>2</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="m_llColontitleBottom">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>19</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bottom</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="m_llColontitleTop">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>20</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Top</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>130</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Scale</string>
   </property>
   <widget class="QRadioButton" name="m_rbScalePunctual">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>60</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Punctual</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_leScalePunctual">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>999</string>
    </property>
    <property name="text">
     <string>100</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="cursorPosition">
     <number>3</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_rbScalePunctual2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>114</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>width of the sheet</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>m_cbPaperSize</tabstop>
  <tabstop>m_rbPortrait</tabstop>
  <tabstop>m_rbLandscape</tabstop>
  <tabstop>m_leFieldsLeft</tabstop>
  <tabstop>m_leFieldsTop</tabstop>
  <tabstop>m_leFieldsRight</tabstop>
  <tabstop>m_leFieldsBottom</tabstop>
  <tabstop>m_leRTitleTop</tabstop>
  <tabstop>m_leRTitleBottom</tabstop>
  <tabstop>m_rbScalePunctual</tabstop>
  <tabstop>m_leScalePunctual</tabstop>
  <tabstop>m_rbScalePunctual2</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="uoReport.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>uorPageSetup</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>uorPageSetup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
